tests/random_test_gym.py

```diff
@@ -189,7 +189,7 @@
                                    - n_rounds * (max_continue + 1) * env.max_delta_health,
                                    np.mean(cumulative_ep_rew_all)))
 
-        print("ALL GOOD!")
+        print("COMPLETED SUCCESSFULLY!")
     except Exception as e:
```

Comment:
Why do we catch the exception at all? Wouldn't it make sense to just have it terminate the script with non-zero return code instead?

In general we should use a proper testing library but let's address that in some futher iteration.

Reply:
Yes we do need a proper testing suite, this is how it works for now, let's keep this in mind for the proper iteration in which we will put in place a complete testing functionality